dataservice/api/genomic_file/models.py

```diff
@@ -55,6 +56,12 @@ class GenomicFile(db.Model, Base, IndexdFile):
     biospecimen_id = db.Column(KfId(), db.ForeignKey('biospecimen.kf_id'),
                                nullable=True)
 
+    read_group = db.relationship(ReadGroup,
```

Comment:
First want to make sure I understand the relationship between genomic file and read group properly. From Yuankun's description in #343, sounds like there is a 1 genomic_file --> many read_groups relationship? If so, then this attribute should be `read_groups`.

Reply:
That's correct - there's often many read groups that get combined into one file.

Reply:
I don't believe the intent here is to associate all read groups to their resulting bam but rather to link any read group metadata a fastq has back to it. So I'm not sure we will be storing relationships between bam genomic files and read groups.

Reply:
I'm revisiting this PR bc I see a couple issues. I'm still confused on the relationship here. Is there a 1-1 relationship between genomic file and read group or is it 1 to many? If it is 1 to 1, then you probably want to add `uselist=False` parameter to the relationship. However, if it really is 1-1, then why not make these fields part of genomic file entity? If it is NOT 1-1, then this attribute really should be named `read_groups`. It looks like this might just be a typo since everywhere else in the code the attribute is referred to as `read_groups`.

Reply:
Aligned Reads are compiled from fastq files divided by Read Group, so the relation is 1-1. Not all files are fastq with information about the Read Group, which is why it is preferable to store in a different table.